django/db/models/fields/__init__.py

```diff
@@ -1812,14 +1812,27 @@ def _check_max_length_warning(self):
     def validators(self):
         # These validators can't be added at field initialization time since
         # they're based on values retrieved from `connection`.
+        validators_existed = super(IntegerField, self).validators
         range_validators = []
```

Comment:
As you'll already be storing validators in `validators` you can remove this variable.